ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/server/api/services/grpc/GrpcCommandService.scala

```diff
@@ -19,6 +21,8 @@ import scala.concurrent.Future
 class GrpcCommandService(
     protected val service: CommandService with AutoCloseable,
     val ledgerId: LedgerId,
+    currentTime: () => Instant,
+    maxTtl: () => Duration
```

Comment:
Is the goal to be able to update TTL on-the-fly after this service has been constructed?

Reply:
The maximum TTL is a ledger configuration parameter. The ledger configuration can change at any time, with changes communicated via the `ReadService`.

Reply:
Thanks for the confirmation